shared/AppInsightsCore/Tests/Unit/src/ApplicationInsightsCore.Tests.ts

```diff
@@ -350,7 +350,7 @@ export class ApplicationInsightsCoreTests extends AITestClass {
 
         // TODO: test stopPollingInternalLogs
         this.testCase({
-            name: "DiagnosticLogger: stop Polling InternalLogs",
+            name: "DiagnosticLogger: stop Polling InternalLogs flushes logs when not empty",
```

Comment:
This is a minor change to the functionality, but generally this is only called internally.
As part of the upcoming "unload" feature stopping needs to flush which is why the change is occurring.